applications/server/src/store/host-resolvers.ts

```diff
@@ -31,7 +31,7 @@ export class HostResolvers {
 		@Arg('hostDisplayName', () => String) hostDisplayName: string,
 		@Arg('shape', { nullable: true }) shape?: Shapes,
 		@Arg('color', { nullable: true }) color?: string,
-		@RelationPath() relationPaths: Relation<Host>
+		@RelationPath() relationPaths?: Relation<Host>
```

Comment:
@GoldingAustin, This was giving me an error that `A required parameter cannot follow an optional parameter. ts(1016)` so I made it optional?